apps/api/src/app/execution-details/dtos/execution-details-response.dto.ts

```diff
@@ -27,16 +26,16 @@ export class ExecutionDetailsResponseDto {
   @ApiPropertyOptional()
   _messageId?: string;
 
-  @ApiProperty()
-  providerId: string;
+  @ApiPropertyOptional()
+  providerId?: string;
 
   @ApiProperty()
   transactionId: string;
 
   @ApiProperty({
-    enum: ChannelTypeEnum,
+    enum: StepTypeEnum,
   })
-  channel: ChannelTypeEnum;
+  channel?: StepTypeEnum;
```

Comment:
I do not get this change. Does step type holds channel information?
What's the different between `StepType` and `ChannelType`?

Reply:
StepType contains digest and delay as well ChannelType only contains types of providers